package.json

```diff
@@ -41,11 +41,12 @@
   },
   "devDependencies": {
     "babel": "^5.8.12",
-    "babel-eslint": "^4.0.5",
+    "babel-eslint": "^7.0.0",
     "babel-loader": "^5.3.2",
-    "eslint": "^1.9.0",
+    "eslint": "^3.7.1",
+    "eslint-config-buildo": "buildo/eslint-config",
     "eslint-loader": "^1.1.1",
-    "eslint-plugin-react": "^3.8.0",
+    "eslint-plugin-react": "^5",
```

Comment:
this should be removed as it's already a dependency of `buildo-eslint-config` (https://github.com/buildo/eslint-config/blob/master/package.json#L29)